theano/gpuarray/c_code/dnn_fwd.c

```diff
@@ -167,31 +204,17 @@ APPLY_SPECIFIC(conv_fwd)(PyGpuArrayObject *input, PyGpuArrayObject *kerns,
             cuda_exit(c->ctx);
             return 1;
           }
+          prev_algo.algo = algo;
+          // no tensor_op returned from Get()
```

Comment:
Yes it is set here: 
  // if FindEx was used (choose_time), workspace size is set.                                                                                                                                                                                                   
  if (!(reuse_algo || use_cached || params->choose_time))